### PR TITLE
437341 sonarcloud fixes 3

### PR DESCRIPTION
**Pull request by flurdy**

### CODE REVIEW

File: Backend.Api/Utils/Mongo/MongoDbClientFactory.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ namespace Backend.Api.Utils.Mongo;
 public class MongoDbClientFactory : IMongoDbClientFactory
 {
     private readonly IMongoDatabase _mongoDatabase;
-    private readonly IMongoClient _client;
+   private readonly MongoClient _client;
```
Comment (christopherjturner) [inline Backend.Api/Utils/Mongo/MongoDbClientFactory.cs:12]: is this change intended?

Comment (flurdy) [reply Backend.Api/Utils/Mongo/MongoDbClientFactory.cs:12]: SonarCloud moaned about inefficiency using an interface...

**Review by christopherjturner — APPROVED** [review]
